### ISSUE TITLE
Can't create an integration with region SE3 and SEK

### ISSUE THREAD

**Issue by nilaz** [custom-components/nordpool#107]
## Version of the custom_component
0.0.4

## Homeassistant version
Home Assistant 2021.12.9

## Configuration
Using the ui wizard with the following input:
![image](https://user-images.githubusercontent.com/1112093/149362363-b7578f96-ab2c-4d41-a1f8-c4baf89e2dc8.png)

Logging set to
```yaml
logger:
  default: warning
  logs:
    homeassistent.components.mqtt: debug
    pydeconz: debug
    homeassistant.components.deconz: debug
    pymodbus: debug
    homeassistant.components.modbus: debug
    pymodbus.client: debug
    nordpool: debug
    custom_components.nordpool: debug
    custom_components.nordpool.sensor: debug
    custom_components.nordpool.aio_price: debug
```

## Describe the bug
I want to create an integration with region SE3 and currency SEK. I fill in the wizard as shown above. The error is seen in the picture.
It says I should use one of the listed currencies. I have now tried all four mentioned in the message and there is an integration created for the three other currencies in the list, but it does not work for the last one that I would like to use, SEK.
When I look at the three created integrations I see values read for all three of them. The sensors seem to work as they should.

Edit: I have been running Nordpool for some time but since the last update I have not seen anything in my graph. I don't know exactly when it stopped working.

## Debug log

I see nothing in the log

**Comment by nilaz** [comment]
I agree it looks greyed out but I can still chose any of the four currencies in the list in the error message.
One good thing and one bad thing:

- When I tried again to test what you wrote above I test region SE4 and it worked well but that is not where I live. 
- I spent some time trying to understand how to use the dev tools to see what the http call looked like. I understand from the documentation that I should start listen to events and then try integrate and hit save button. I don't understand what I should start listen to. Any suggestion?

**Comment by ErikH2** [comment]
I think the entry "1" in lägsta prisnivå is not allowed, try 0.95 or below 1

**Comment by nilaz** [comment]
Ok, I can test later today. "1" was the default value. I did not really know what it meant in the UI.

**Comment by nilaz** [comment]
I tested with a couple of values lower than 1. Did not change anything for me. I still can't create an integration with SEK. It worked great with DKK today...
I still don't know what to listen to in the dev tools as @Hellowlol proposed earlier. I really want to help out but I don't understand what to do to listen to the http call.

**Comment by nilaz** [comment]
Now I'm blushing. You are correct about me not choosing any currency. Since it looked like the default value is set to SEK I did not select it again. When I did select it on purpose it worked directly.
I'm very sorry for me not understanding yet earlier.

However, I'm proposing then that there is no value shown at all in the popup so that users like me has to chose a currency. Or is it obvious for everyone else?

It works for me now anyway.
